.github/workflows/codegen.yml

```diff
@@ -10,11 +10,10 @@ jobs:
 
     - uses: actions/setup-go@v2
       with:
-        go-version: 1.13
+        go-version: 1.15
```

Comment:
```suggestion
        go-version: 1.16
```

?

Reply:
Even in master some actions are running on 1.15 and some on 1.16 since both are in the support list, so I just moved these to 1.15